querybook/server/datasources/datadoc.py

```diff
@@ -327,6 +328,8 @@ def update_datadoc_schedule(id, cron=None, enabled=None, kwargs=None):
             updated_fields["cron"] = cron
         if enabled is not None:
             updated_fields["enabled"] = enabled
+            if enabled and not schedule.enabled:
```

Comment:
thanks for the fix! can you add a comment here explaining why it is needed?

Reply:
Without resetting `last_run_at` whenever a previously running schedule is reenabled, the datadoc runs immediately when it is reenabled instead of the next scheduled time. 

For example, in the following situation:
- `last_run_at` = 3/1/23 at 9am
- Schedule was reenabled at 3/3/23 at 10:15am
- Datadoc was scheduled to run every hour

Based on the current qb logic the scheduler runs periodically to see if it is time to start a run. Since several hours have passed between `last_run_at` and now (the time the schedule was reenabled), and the expected schedule was for the datadoc to run every hour, the datadoc runs as soon as the schedule is enabled again instead of waiting to run at the next expected time. This results in unexpected behavior since in this scenario, the datadoc runs at 3/3/23 at 10:15am instead of 3/3/23 at 11am.

Reply:
hey @lilyli9, just to clarify I understand the context on the fix. What I meant is that can you leave a 1 line comment in the code to explain that updating `last_run_at` is needed so it doesn't not retrigger a new run when changing the scheduled datadoc from disabled to enabled

Reply:
Thanks for the clarification, I added a comment.